src/flyout.js

```diff
@@ -174,30 +167,35 @@ export class FlyoutElement extends LitElement {
       <dl>
         <dt>On Read the Docs</dt>
         <dd>
-          <a href="${this.getProjectUrl()}">Project Home</a>
+          <a href="${this.config.projects.current.urls.home}">Project Home</a>
         </dd>
         <dd>
-          <a href="${this.getProjectUrl()}builds/">Builds</a>
+          <a href="${this.config.projects.current.urls.builds}">Builds</a>
         </dd>
         <dd>
-          <a href="${this.getProjectUrl()}downloads/">Downloads</a>
+          <a href="${this.config.projects.current.urls.downloads}">Downloads</a>
         </dd>
       </dl>
     `;
   }
 
   renderDownloads() {
-    if (!this.config.addons.flyout.downloads.length) {
+    if (!Object.keys(this.config.versions.current.downloads).length) {
       return nothing;
     }
 
+    const nameDisplay = {
+      pdf: "PDF",
+      epub: "ePUB",
```

Comment:
Maybe either are fine, but [the specification](https://www.w3.org/AudioVideo/ebook/) uses this:

```suggestion
      epub: "EPUB",
```